runelite-client/src/main/java/net/runelite/client/plugins/poh/PohPlugin.java

```diff
@@ -64,6 +68,9 @@
 	@Getter(AccessLevel.PACKAGE)
 	private final Map<GameObject, Tile> pohObjects = new HashMap<>();
 
+	@Getter(AccessLevel.PACKAGE)
+	private DecorativeObject gloryMount;
```

Comment:
Any reason why this isn't a map<deco object, tile>?

Reply:
I did not add a map since the glory mount was the only decorative object we have currently, but I re-added a map for it in case we ever need to add another decorative object.